src/logic/schedule-parser/schedule.parser.ts

```diff
@@ -32,58 +32,57 @@ export class ScheduleParser implements ScheduleProvider {
   }
 
   private parseSections(rawSchedule: string[][][]): Sections {
-    let children;
     let nurses;
-    let babysiter;
-    let dayWorkers;
+    let babysiters;
 
-    const metadataRaw = rawSchedule.find((r) => r[0][0].toLowerCase().trim() === "dni miesiąca");
+    const foundationInfoHeaders = ["dni miesiąca", "pracownicy dzienni", "dzieci"];
+
+    const foundationInfoRaw = rawSchedule.find((r) =>
+      foundationInfoHeaders.some((h) => r[0][0].toLowerCase().trim() === h)
+    );
+    const metadataRaw = foundationInfoRaw?.find(
+      (r) => r[0].toLowerCase().trim() === "dni miesiąca"
+    );
     const metadata = new MetaDataParser(this.month, this.year, metadataRaw);
 
+    const childrenRaw = foundationInfoRaw?.find((r) => r[0].toLowerCase().trim() === "dzieci");
+    const children = new ChildrenInfoParser(metadata, childrenRaw);
+
+    const extraWorkersRaw = foundationInfoRaw?.find(
+      (r) => r[0].toLowerCase().trim() === "pracownicy dzienni"
+    );
+    const extraWorkers = new ExtraWorkersInfoParser(metadata, extraWorkersRaw);
+
+    rawSchedule.splice(rawSchedule.indexOf(foundationInfoRaw!), 1);
+
     rawSchedule.forEach((r) => {
-      if (r && r.length !== 0 && r[0].length !== 0) {
-        const rKey = r[0][0].toLowerCase().trim();
-        if (rKey === "dni miesiąca") {
-          if (r.length === 3) {
-            children = new ChildrenInfoParser(metadata, r);
-            dayWorkers = new ExtraWorkersInfoParser(metadata, r);
-          }
-        } else if (rKey === "pracownicy dzienni") {
-          dayWorkers = new ExtraWorkersInfoParser(metadata, r);
-        } else if (rKey === "dzieci") {
-          children = new ChildrenInfoParser(metadata, r);
-        } else if (!nurses) {
+      if (r) {
+        if (!nurses) {
           nurses = new ShiftsInfoParser(WorkerType.NURSE, metadata, r);
         } else {
```

Comment:
```suggestion
        } else if (!babysiters) {
```

Reply:
@kaplonpaulina 
Można to komitować?

Wygląda sensownie, ale mi chodzi o to dlaczego ty tego  nie skomitowałaś od razu, a dałaś jako suggestion?